front/src/modules/ui/components/editable-cell/types/EditableRelation.tsx

```diff
@@ -119,7 +119,7 @@ export function EditableRelation<RelationType, ChipComponentPropsType>({
 
   // TODO: Tie this to a react context
   const [filterSearchResults, setSearchInput, setFilterSearch, searchInput] =
-    useSearch<RelationType>();
+    useSearch<RelationType>({});
```

Comment:
@lucasbordeau If the field `currentSelectedId` is optional, we should also put the object as optional, so the function param can be empty.

Reply:
Ok done